src/course-unit/data/api.js

```diff
@@ -148,3 +149,31 @@ export async function duplicateUnitItem(itemId, XBlockId) {
 
   return data;
 }
+
+/**
+ * Get an object containing course outline data.
+ * @param {string} courseId - The identifier of the course.
+ * @returns {Promise<Object>}
+ */
+export async function getCourseOutlineInfo(courseId) {
+  const { data } = await getAuthenticatedHttpClient()
+    .get(getCourseOutlineInfoUrl(courseId));
+
+  return camelCaseObject(data);
+}
+
+/**
+ * Move a unit item to new unit.
+ * @param {string} sourceLocator - The ID of the item to be moved.
+ * @param {string} targetParentLocator - The ID of the XBlock associated with the item.
+ * @returns {Promise<Object>} - A promise that resolves to the response data from the server.
```

Comment:
Would it be possible for you to specify the actual shape of the data returned, instead of just `Object` ?

Reply:
Added